### PR TITLE
Java homework 3

### PR DESCRIPTION
**Pull request by hospikos**

### CODE REVIEW

File: src/main/java/homework3/Task1.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package homework3;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int a = 13;
+        float b = 6.55f;
```
Comment (alexshamrai) [inline src/main/java/homework3/Task1.java]: "Create a program which has two integer numeric variables "
```suggestion
        int b = 6;
```

File: src/main/java/homework3/Task1.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        float SumResult = a+b;
```
Comment (alexshamrai) [inline src/main/java/homework3/Task1.java]: variables names should start with lowercase letter
```suggestion
        float sumResult = a + b;
```

File: src/main/java/homework3/Task1.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        float SubtractionResult = a-b;
```
Comment (alexshamrai) [inline src/main/java/homework3/Task1.java]: Keep spaces between operator and operands
```suggestion
        float subtractionResult = a - b;
```
Or use "Format Code" Intelij IDE feature

File: src/main/java/homework3/Task4.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package homework3;
+
+public class Task4 {
+    public static void main(String[] args) {
+        int a = 11;
+        int b = 2;
+        int Remainder = a%b;
```
Comment (alexshamrai) [inline src/main/java/homework3/Task4.java]: same here, variable naming and spaces(format code)

File: src/main/java/homework3/Task5.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package homework3;
+
+public class Task5 {
+    public static void main(String[] args) {
+        double Celsium = 5;
```
Comment (alexshamrai) [inline src/main/java/homework3/Task5.java]: fix variables naming

File: src/main/java/homework3/Task5.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+        double Fahrenheit = Celsium * (9/5) + 32;
```
Comment (alexshamrai) [inline src/main/java/homework3/Task5.java]: Fahrenheit calculation is incorrect
5 C -> 41 F
In your case it is 37. Think why

**Review by alexshamrai — CHANGES_REQUESTED** [review]

**Comment by hospikos** [comment]
@alexshamrai done

**Review by alexshamrai — APPROVED** [review]

**Comment by alexshamrai** [comment]
Way to go!

**Pull request merged**